src/config/index.ts

```diff
@@ -24,9 +18,31 @@ export const getJSONEnv = (key: string) => {
   return JSON.parse(value);
 };
 
-const config: Config = {
-  appEnv: getEnv('APP_ENV'),
+const configObject = {
+  appEnv: process.env.APP_ENV,
+  serverPort: Number(process.env.PORT || process.env.SERVER_PORT),
```

Comment:
Could be NaN. Might be better to do the parsing and conversion with `zod.string().transform(/*make sure it is a valid port and throw otherwise*/)